### PR TITLE
Update setup.asciidoc

### PR DESCRIPTION
**Pull request by markwalkom**
Added a comment explicitly noting that Java 9 is supported from 6.2 onwards.

### CODE REVIEW

**Comment by jasontedor** [comment]
@markwalkom I would rather that we not do this. The support matrix is the gold standard for what we support, I would rather not maintain this in two places. Additonally, the JDK that we support will evolve rapidly as non-LTS releases come and go, this will be cumbersome to maintain (a simple matrix showing version/JVM breakdown is easier to understand than the carefully worded statements we would need to provide here). Lastly, the existing wording here is intentional: we want to encourage users to stay on LTS releases, or at least think very carefully before they migrate to a non-LTS release. For this reason I would prefer to not give prominence to JDK 9.

To summarize: let’s maintain this information in one place (the support matrix) and keep the existing wording.

**Comment by markwalkom** [comment]
Righto, can we add a link to the matrix then?

**Comment by jasontedor** [comment]
There already is one? Check the highlighted text “[supported](https://www.elastic.co/support/matrix)” from this [page](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/setup.html).

**Comment by markwalkom** [comment]
:facepalm:

**Comment by jasontedor** [comment]
No worries!
